packages/loader/container-loader/src/loadPaused.ts

```diff
@@ -35,7 +36,7 @@ import { GenericError } from "@fluidframework/telemetry-utils/internal";
  *    network connectivity issues / ability to cancel (IContainer.disconnect) or close container (IContainer.close)
  *    This flow needs to fetch ops (potentially connecting to delta connection), and any retriable errors on this path result in infinite retry.
  *    If you need to cancel that process, consider supplying AbortSignal parameter.
- * @param loader - loader instance to use to load container
+ * @param loaderProps - loader props to use to load container.
```

Comment:
Nit (since we're already making a change here)
```suggestion
 * @param loaderProps - The loader props to use to load the container.
```